src/Button/Button.tsx

```diff
@@ -9,14 +9,18 @@ import React from "react";
 import useStyles from "./styles";
 
 export type ButtonVariant = "primary" | "secondary" | "tertiary";
+// Aliasing "default" to "text" because "default" isn't actually default
```

Comment:
Can you elaborate? Color of type "text" is not saying much to me

Reply:
Black in light mode, white in dark mode. `textPrimary` is used in typographies, though we won't have `textSecondary` in buttons. `textColor` would be doing something like this: `color="textColor"` so it's not so cool too. I don't really have any idea how to make it clearer other than naming this color `text`

Reply:
Alright I see, our default is `color="primary"`